tests/rule_based_profiler/test_profiler_user_workflows.py

```diff
@@ -87,6 +87,7 @@ def test_alice_profiler_user_workflow_single_batch(
         expectation_suite_name=alice_columnar_table_single_batch[
             "expected_expectation_suite_name"
         ],
+        include_citation=False,
```

Comment:
Fixtures do not contain metadata. As `include_citation' defaults to `True`, should we update the fixtures to include the appropriate metadata? I've set `include_citation` to `False` here in order to pass tests and tested the citation capabilities in `test_profile_includes_citations` and `test_profile_excludes_citations`.

Reply:
@cdkini The metadata content is created by the `parameter_builder` actions and stored into `ExpectationConfiguration` by the `expectation_configuration_builder` classes.  We should test with `include_citation` set to`True` in order to exercises the full (default) functionality.  Happy to discuss if something does not seem right.  Thanks.

Reply:
Agreed!

Reply:
Have updated both `bobby` and `alice` fixtures to include appropriate citations.